src/transpiler.ts

```diff
@@ -249,51 +327,67 @@ function getStatementsToAppend(program: es.Program): es.Statement[] {
 
 function splitLastStatementIntoStorageOfResultAndAccessorPair(
   lastStatement: es.Statement
-): es.Statement[] {
+): [es.Statement, es.Statement, {}?] {
```

Comment:
Same as above, I didn't notice this before but consider using an object instead. Object literal shorthands and destructuring make it quite easy to return and assign these. Objects are more "type safe" compared to arrays as there are names going along with each field.

Reply:
Agree.